charts/stable/rtorrent-rutorrent/values.yaml

```diff
@@ -50,20 +50,45 @@ service:
     ports:
       rtinc:
         enabled: true
-        port: 50000
+        port: 50000      
+  phphealth:
+    enabled: true
+    ports:
+      phphealth:
+        enabled: true
+        port: 8081
+        
 
 workload:
   main:
     podSpec:
       containers:
         main:
+          probes:
+            liveness:
+              enabled: true
+              type: http
+              path: /ping
+              port: 8081
+            readiness:
+              enabled: true
+              type: http
+              path: /ping
+              port: 8081
+            startup:
+              enabled: true
```

Comment:
Enabled is not needed in any probe.
Same for type
Startup  also remove path as tcp does not have path